static/schemas/obms/amt-obm-service.json

```diff
@@ -1,9 +1,10 @@
 {
     "title": "amt-obm-service",
     "definitions": {
-        "Obm": {
+        "ObmPatch": {
```

Comment:
@srinia6 I think we want to move the `service` property out of `ObmPatch` and into `Obm`.  That will prevent changing the service type with a patch.